libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/telemetryevents/TelemetryArrival.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.navigation.core.telemetry.telemetryevents
+
+import android.annotation.SuppressLint
+import android.os.Parcel
+import com.mapbox.android.telemetry.Event
+
+/**
+ * Documentation is here [https://paper.dropbox.com/doc/Navigation-Telemetry-Events-V1--AuUz~~~rEVK7iNB3dQ4_tF97Ag-iid3ZImnt4dsW7Z6zC3Lc]
+ */
+
+// Defaulted values are optional
+@SuppressLint("ParcelCreator")
+class TelemetryArrival(val Metadata: TelemetryMetadata) : Event() {
```

Comment:
How's this different from `NavigationArriveEvent`? Why do we need this new implementation?

Reply:
NavigationArriveEvent depends on PhoneState which I am not using and it also depends on NavigationEvent, which I am not using either. Instead, I have a new class, TelemetryMetadata, that does the job of NavigationEvent.